prc/ecdata/trivial_pairing.tex

```diff
@@ -3,6 +3,7 @@
     % \item \If $\internalChecksPassed_{i} = 0$ \Then $\trivialPairing_{i} = 0$
     % \item \If $\notOnGTwoAccMax_{i}      = 1$ \Then $\trivialPairing_{i} = 0$
     \item \trivialPairing{} is \textbf{counter-constant binary} \quad (\trash)
+    \item \If $\isEcpairingResult_{i} = 1$ \Then \trivialPairing{} is \textbf{index-constant} \quad (\trash)
```

Comment:
Yes, but without the `(\trash)`

Reply:
Also I would lead with the `$\locIsEcpairing _{i} = 0$` case. Otherwise it's
- nontrivial case condition
- trivial case condition
- nontrivial case conditions

Which is a little strange.

Reply:
The right order is
```
- \trivialPairing{} is \textbf{counter-constant binary} \quad (\trash)
- \If $\locIsEcpairing _{i} = 0$ \Then $\trivialPairing _{i} = 0$
- \If $\isEcpairingData_{i - 1} = 0$ \et $\isEcpairingData _{i} = 1$ \Then $\trivialPairing_{i} = 1$
- \If $\isEcpairingResult_{i} = 1$ \Then \trivialPairing{} is \textbf{index-constant}
- remaining stuff
```

Reply:
So it's
- global constancy behaviour
- trivial case
- nontrivial case: entry
- nontrivial case: transition
- other stuff
```